### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by dan-jones-iw**

### CODE REVIEW

File: coordinator/README.md [hunk 1]
```diff
@@ -11,11 +11,27 @@ npm install
...
+
 ## Deployment
 
 Redeploy the API Proxy. See the main [README.md](../README.md).
 
 ## Endpoints
 
-- [ ] POST `/ConvertFullMessage`
-- [ ] POST `/ConvertSignatureFragments`
+Endpoints are found in `/src/routes`.
```
Comment (harryhmc) [inline coordinator/README.md:31]: You've not mentioned the `/src` directory in the file paths in the Directories section but you have here. Maybe just write `/routes` for consistency with above?

File: coordinator/README.md [hunk 2]
```diff
@@ -11,11 +11,27 @@ npm install
 npm run serve
 ```
 
+### Directories
+- `/models` Typescript interface/class definitions
+- `/resources` mustache template files for xml responses
+- `/routes` API endpoint definitions
```
Comment (harryhmc) [inline coordinator/README.md:17]: Do `/routes` subdirectories need to be added here, or is that a level of detail we can comfortably leave out?

**Review by harryhmc — COMMENTED** [review]
You'll need to update the `make release` command for this repo -- it's still referencing the terraform directory which is causing the build pipeline to fail

File: coordinator/README.md (hunk 2 above)
Comment (dan-jones-iw) [reply coordinator/README.md:17]: just did it for consistency, gone into more detail below, can delete if u think its pointless

File: coordinator/README.md (hunk 1 above)
Comment (dan-jones-iw) [reply coordinator/README.md:31]: was thinking about putting `/src` in the directories section tbh, maybe that would make this better?

Comment (harryhmc) [reply coordinator/README.md:31]: yh that looks good

**Review by harryhmc — APPROVED** [review]

**Review by harryhmc — APPROVED** [review]

**Review by harryhmc — APPROVED** [review]

**Pull request merged**